### PR TITLE
DOC-798 - Mendable AI Widget Integration

### PR DESCRIPTION
**Pull request by nagesh161007**
## Describe the Change

Add Mendable AI search widget. Added script in the `gatsby-ssr.js` similar to full story and appzi integration and initialize the widget in the persistant layout useEffect so it will not destroyed during navigation

https://www.gatsbyjs.com/docs/how-to/routing/layout-components/

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-798)

### CODE REVIEW

File: src/shared/components/MendableAIWidget.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    document.body.appendChild(script);
+
+    script.onload = () => {
+      if (window.Mendable) {
+        window.Mendable.initialize({
+          anon_key: process.env.GATSBY_MENDABLE_API_KEY,
+          type: "floatingButton",
+          dialogPlaceholder: "What is Palette?",
+          floatingButtonStyle: {
+            color: "#fff",
```
Comment (karl-cardenas-coding) [inline src/shared/components/MendableAIWidget.js:21]: Should this be `#ffffff` instead?

**Review by karl-cardenas-coding — APPROVED** [review]
Looks great, just a minor question.

File: src/shared/components/MendableAIWidget.js (hunk 1 above)
Comment (nagesh161007) [reply src/shared/components/MendableAIWidget.js]: Updated

File: src/shared/components/MendableAIWidget.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "https://unpkg.com/@mendable/search@0.0.109/dist/umd/mendable-bundle.min.js";
+
+export default function MendableAIWidget() {
+  useEffect(() => {
+    const script = document.createElement("script");
+    script.src = MENDABLE_SCRIPT_URL;
+    script.async = true;
+
+    document.body.appendChild(script);
+
```
Comment (karl-cardenas-coding) [inline src/shared/components/MendableAIWidget.js:13]:
```suggestion
```

**Comment by karl-cardenas-coding** [comment]
Added a commit to retrigger the CI. Looks like it got stuck after the image compression job 🤔

**Pull request merged**